tests/k8s_testcases/expected_output/semantic_diff_online_boutique_new_vs_snythesized_new.txt

```diff
@@ -0,0 +1,13 @@
+new_online_boutique and new_online_synthesis_res are not semantically equivalent.
+Changed connections between persistent peers:
+Removed connections (based on topology from config: new_online_boutique) :
+src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [loadgenerator] conn: All connections
+
+Changed connections between persistent peers and ipBlocks:
+Removed connections (based on topology from config: new_online_boutique) :
+src ip block: 0.0.0.0/0 dst_ns: [default] dst_pods: [loadgenerator] conn: All connections
+src ip block: ::/0 dst_ns: [default] dst_pods: [loadgenerator] conn: All connections
+src_ns: [default] src_pods: [*] dst ip block: 0.0.0.0/0 conn: All connections
+src_ns: [default] src_pods: [*] dst ip block: ::/0 conn: All connections
+
+
```

Comment:
Typo in file name: snythesized -> synthesized

Reply:
done